rpi_lcd/__init__.py

```diff
@@ -39,8 +42,12 @@ def _write_byte(self, byte):
         sleep(self.delay)
 
     def write(self, byte, mode=0):
-        self._write_byte(mode | (byte & 0xF0) | 0x08)
-        self._write_byte(mode | ((byte << 4) & 0xF0) | 0x08)
+        if self.backlight_status == True:
+            backlight_value = LCD_BACKLIGHT
+        else:
+            backlight_value = LCD_NOBACKLIGHT
```

Comment:
Hey, what would you say to simplify the code by:

```suggestion
        backlight_mode = LCD_BACKLIGHT if self.backlight_status else LCD_NOBACKLIGHT
```

Reply:
ok looks good
```
    def write(self, byte, mode=0):
        backlight_mode = LCD_BACKLIGHT if self.backlight_status else LCD_NOBACKLIGHT
        self._write_byte(mode | (byte & 0xF0) | backlight_mode)
        self._write_byte(mode | ((byte << 4) & 0xF0) | backlight_mode)
```